docs/guide/cli/cli_yaml.rst

```diff
@@ -2,9 +2,10 @@ Customising CLI planning with yaml settings
 ===========================================
 
 The planning commands in the CLI can be made more powerful by supplying
-"yaml" formatted files to customise the planning algorithms.
-This settings file has a series of sections for customising the different algorithms,
-as an example, the settings file which re-specifies the default behaviour would look like ::
+yaml-formatted files to customise the planning algorithms.
```

Comment:
```suggestion
YAML-formatted files to customise the planning algorithms.
```
I believe it's meant to be capitalized so we probably should do that when we talk about the "thing".

Reply:
agreed